File: Modul 6/py/PRAK605-2310817110007-MuhammadDaffaMusyafa.py
import numpy as np

ordo = int(input())

matriksA = np.zeros((ordo, ordo), dtype=int)
print("Matriks A")
for i in range(ordo):
    matriksA[i] = list(map(int, input().split()))

matriksB = np.zeros((ordo, ordo), dtype=int)
print("Matriks B")
for i in range(ordo):
    matriksB[i] = list(map(int, input().split()))

hasil_kali = np.dot(matriksA, matriksB)

print("Matriks AxB")
for row in hasil_kali:
    print(" ".join(map(str, row)))

File: Modul 6/C/PRAK605-2310817110007-MuhammadDaffaMusyafa.c
#include <stdio.h>

int main() {
    int matriksA[10][10], matriksB[10][10], hasil_kali[10][10];
    int ordo, hasil = 0;

    scanf("%d", &ordo);

    printf("Matriks A\n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            scanf("%d", &matriksA[i][j]);
        }
    }

    printf("Matriks B\n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            scanf("%d", &matriksB[i][j]);
        }
    }

    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            for (int k = 0; k < ordo; k++) {
                hasil += matriksA[i][k] * matriksB[k][j];
            }
            hasil_kali[i][j] = hasil;
            hasil = 0;
        }
    }

    printf("Matriks AxB\n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            printf("%d ", hasil_kali[i][j]);
        }
        printf("\n");
    }

    return 0;
}
